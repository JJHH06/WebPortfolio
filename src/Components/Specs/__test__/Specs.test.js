/**
 * @jest-environment jsdom
 */
/* eslint-disable*/
import React from 'react';
import ReactDOM from 'react-dom';
import { cleanup } from '@testing-library/react';
import { isTSanyKeyword } from '@babel/types';
import Specs from '../Specs.jsx';
import {render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from "react-test-renderer";


afterEach(cleanup);
it('renders footer correctly', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Specs />, div);
});

it("Renders footer text correctly",()=>{
    const {getByTestId} = render(<Specs/>)
    expect(getByTestId('progress-text')).toHaveTextContent("Java:")
})

it("matches snapshot",()=>{
    const tree = renderer.create(<Specs/>).toJSON()
    expect(tree).toMatchSnapshot();
})