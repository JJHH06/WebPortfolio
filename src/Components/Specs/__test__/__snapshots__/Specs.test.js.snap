// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container seccion-portafolio"
  id="curriculum"
>
  <div
    className="row texto-about"
    data-aos="fade"
  >
    <div
      className="col-lg-8  mx-auto"
    >
      <h2
        className="display-5"
      >
        <b>
          Logros y Habilidades
        </b>
      </h2>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="d-flex align-items-center col mx-auto list-group"
    >
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-right"
      >
        <div
          className="d-flex w-100 justify-content-between"
        >
          <h5
            className="mb-1"
          >
            Galardon académico de la Riva
          </h5>
          <small
            className="text-muted"
          >
            2018
          </small>
        </div>
        <p
          className="mb-1"
        >
          Premio a los mejores promedios de la promoción de colegio.
        </p>
        <small
          className="text-muted"
        >
          Obtenido mientras estudié Bachillerato en Computación
        </small>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-right"
      >
        <div
          className="d-flex w-100 justify-content-between"
        >
          <h5
            className="mb-1"
          >
            Ingeniería en Ciencias de la Computación
          </h5>
          <small
            className="text-muted"
          >
            2019-2023
          </small>
        </div>
        <p
          className="mb-1"
        >
          Universidad del Valle de Guatemala
        </p>
        <small
          className="text-muted"
        >
          Actualmente estudiando y especializandome en seguridad informática
        </small>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-right"
      >
        <div
          className="d-flex w-100 justify-content-between"
        >
          <h5
            className="mb-1"
          >
            Beca Fundación Educación
          </h5>
          <small
            className="text-muted"
          >
            2020
          </small>
        </div>
        <p
          className="mb-1"
        >
          Dada por la organización suiza Fundación educación
           
        </p>
        <small
          className="text-muted"
        >
          Beca completa dada a alumnos destacados en carreras tecnológicas
        </small>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-right"
      >
        <div
          className="d-flex w-100 justify-content-between"
        >
          <h5
            className="mb-1"
          >
            Distinción Académica de Estudiante Distinguido
          </h5>
          <small
            className="text-muted"
          >
            2020
          </small>
        </div>
        <p
          className="mb-1"
        >
          Universidad del Valle de Guatemala
        </p>
        <small
          className="text-muted"
        >
          Obtenida por promedio superior a 85 en el año 2019
        </small>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-right"
      >
        <div
          className="d-flex w-100 justify-content-between"
        >
          <h5
            className="mb-1"
          >
            Auxiliar Algoritmos y Programación Básica
          </h5>
          <small
            className="text-muted"
          >
            2021
          </small>
        </div>
        <p
          className="mb-1"
        >
          Universidad del Valle de Guatemala
        </p>
        <small
          className="text-muted"
        >
          Profesor Auxiliar presencial del curso introductorio de programación
        </small>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-right"
      >
        <div
          className="d-flex w-100 justify-content-between"
        >
          <h5
            className="mb-1"
          >
            Distinción Académica Honor al mérito académico
          </h5>
          <small
            className="text-muted"
          >
            2021
          </small>
        </div>
        <p
          className="mb-1"
        >
          Universidad del Valle de Guatemala
        </p>
        <small
          className="text-muted"
        >
          Obtenida por promedio superior a 90 en el año 2020
        </small>
      </a>
    </div>
    <div
      className="d-flex align-items-center col mx-auto list-group"
    >
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-left"
      >
        <p
          className="mb-1"
        >
          Python:
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax="100"
            aria-valuemin="0"
            aria-valuenow="100"
            className="progress-bar progress-bar100"
            role="progressbar"
          />
        </div>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-left"
      >
        <p
          className="mb-1"
          data-testid="progress-text"
        >
          Java:
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax="100"
            aria-valuemin="0"
            aria-valuenow="100"
            className="progress-bar progress-bar100"
            role="progressbar"
          />
        </div>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-left"
      >
        <p
          className="mb-1"
        >
          React:
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax="100"
            aria-valuemin="0"
            aria-valuenow="100"
            className="progress-bar progress-bar75"
            role="progressbar"
          />
        </div>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-left"
      >
        <p
          className="mb-1"
        >
          Javascript:
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax="100"
            aria-valuemin="0"
            aria-valuenow="100"
            className="progress-bar progress-bar85"
            role="progressbar"
          />
        </div>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-left"
      >
        <p
          className="mb-1"
        >
          Flutter:
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax="100"
            aria-valuemin="0"
            aria-valuenow="55"
            className="progress-bar  progress-bar50"
            role="progressbar"
          />
        </div>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-left"
      >
        <p
          className="mb-1"
        >
          C++:
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax="100"
            aria-valuemin="0"
            aria-valuenow="70"
            className="progress-bar progress-bar75"
            role="progressbar"
          />
        </div>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-left"
      >
        <p
          className="mb-1"
        >
          C#:
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax="100"
            aria-valuemin="0"
            aria-valuenow="80"
            className="progress-bar progress-bar70"
            role="progressbar"
          />
        </div>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-left"
      >
        <p
          className="mb-1"
        >
          Kotlin:
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax="100"
            aria-valuemin="0"
            aria-valuenow="55"
            className="progress-bar progress-bar85"
            role="progressbar"
          />
        </div>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-left"
      >
        <p
          className="mb-1"
        >
          Express:
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax="100"
            aria-valuemin="0"
            aria-valuenow="60"
            className="progress-bar progress-bar80"
            role="progressbar"
          />
        </div>
      </a>
      <a
        className="list-group-item list-group-item-action"
        data-aos="fade-left"
      >
        <p
          className="mb-1"
        >
          SQL:
        </p>
        <div
          className="progress"
        >
          <div
            aria-valuemax="100"
            aria-valuemin="0"
            aria-valuenow="100"
            className="progress-bar progress-bar100"
            role="progressbar"
          />
        </div>
      </a>
    </div>
  </div>
</div>
`;
